src/pages/podcast/[shortcode]/[[...slug]].js

```diff
@@ -202,7 +202,7 @@ export default function Episode({ episode, isShortLink }) {
             <meta name="og:description" property="og:description" content={episode.description} />
             <meta name="og:image" property="og:image" content={episode.image} />
 
-            <meta name="twitter:card" content="summary" />
+            <meta name="twitter:card" content="summary_large_image" />
```

Comment:
Not sure if this is the right choice. See #64.

Reply:
Yep, that should be correct!